acceptance/lib/puppet/acceptance/common_utils.rb

```diff
@@ -46,6 +46,14 @@ def setup(agent, o={})
                                      default: {$cron ='cron'} }
                                      package {'cron': name=> $cron, ensure=>present, }])
       end
+
+      # Returns all of the lines that correspond to crontab entries
+      # on the agent. For simplicity, these are any lines that do
+      # not begin with '#'.
+      def crontab_entries_of(host, username)
+        crontab_contents = run_cron_on(host, :list, username).stdout.chomp
```

Comment:
Maybe consider `strip` instead of `chomp` here, just in case blanks come at the beginning

Reply:
Updated the PR.